README.md

```diff
@@ -0,0 +1,42 @@
+# BeBalance Mobile app
+
+This project is mainly focused on Android platform.
+
+### Technologies
+
+1. [React Native Expo](https://docs.expo.dev/get-started/set-up-your-environment/) — mobile library.
+2. [Redux Toolkit](https://redux-toolkit.js.org/) — state manager.
+
+### Folder Structure
+
+1. assets — static assets (images, fonts);
+2. components — common, reusable UI components;
+3. constants — application-wide constants;
+4. features — main app functionalities;
+5. helpers — utility functions and helper methods;
+6. hooks — custom React hooks;
+7. store — Redux Toolkit state management;
+8. types — common type definitions.
+
+### Install dependencies
+
+1. In the root of the mobile folder.
+
+   ```bash
+   npm install
+   ```
+
+2. Create .env file. Use .env.example as a template.
+
+### Start the mobile app
+
+```bash
+ npx expo start
+```
+
```

Comment:
Also, you can add `npm run prebuild` to create /ios and /android folders  &  `npm run android` to build and run app without Expo Go